src/requestforattestation/RequestForAttestation.ts

```diff
@@ -60,6 +61,18 @@ function getHashRoot(leaves: Uint8Array[]): Uint8Array {
 }
 
 export default class RequestForAttestation implements IRequestForAttestation {
+  /**
+   * [STATIC] Builds an instance of [[RequestForAttestation]], from a simple object with the same properties.
+   * Used for deserialization.
+   *
+   * @param obj - An object built from simple [[Claim]], [[Identity]] and legitimation objects.
+   * @returns  A new [[RequestForAttestation]] `object`.
+   * @example
+   * ```javascript
+   * // create an RequestForAttestation object, so we can call methods on it (`serialized` is a Claim, Identity and legitimation object )
```

Comment:
Maybe this is too long. As I am thinking listing all the serialised object is too much